tf_files/aws/modules/eks/README.md

```diff
@@ -76,6 +76,7 @@ users_policy = "test-commons"
 | domain_test | If ha-proxy a domain to check internet access | string | gen3.io |
 | ha_squid | If enabled, this should be set to true | boolean | false |
 | dual_proxy | If migrating from single to ha, set to true, should not disrrupt connectivity | boolean | false |
+| sns_topic_arn | SNS topic ARN for alerts | string | "arn:aws:sns:us-east-1:433568766270:planx-csoc-alerts-topic" |
 
```

Comment:
For community users do we need to augment our instructions to include the SNS setup?
    https://github.com/uc-cdis/cloud-automation/blob/master/doc/csoc-free-commons-steps.md

Reply:
The SNS topic in question allows access from only our accounts, you have to set permissions right, otherwise everyone might be able to publish stuff.

I guess I can come up with something in the documentation for the comunity folks should they want to move on with the new fluentd version/configuration.